website/src/css/custom.css

```diff
@@ -30,7 +30,7 @@
   --ifm-color-primary-lighter: #0682ff;
   --ifm-color-primary-lightest: #2893ff;
   --ifm-code-font-size: 90%;
-  --ifm-font-family-base: "PingFang SC",Segoe UI,Roboto,Ubuntu,Cantarell,Helvetica,sans-serif,Apple Color Emoji,Segoe UI Emoji,Android Emoji,EmojiSymbols,EmojiOne Mozilla,Twemoji Mozilla,Segoe UI Symbol,Noto Color Emoji;
+  --ifm-font-family-base: "sans-serif", Helvetica, PingFang SC,Segoe UI,Roboto,Ubuntu,Cantarell,Helvetica,sans-serif,Apple Color Emoji,Segoe UI Emoji,Android Emoji,EmojiSymbols,EmojiOne Mozilla,Twemoji Mozilla,Segoe UI Symbol,Noto Color Emoji;
```

Comment:
Is there any purpose in changing the font family? I'm not an expert on UI, so just curious about any problem with the original fonts.

Reply:
@wuchong The main reason is that it just renders better, especially for blog posts and its more familiar to the eye